requirements.txt

```diff
@@ -1,11 +1,2 @@
 -e git+https://github.com/nipy/nipype.git@master#egg=nipype
 -e git+https://github.com/incf/pybids.git@master#egg=pybids
-numpy
-lockfile
-future
-scikit-learn
-matplotlib
-nilearn
-sklearn
-nibabel
-niworkflows>=0.0.3a3
```

Comment:
```
pip install -e .
python setup.py install
```

Will both install those dependencies if they are missing.

Please see [these recommendations](https://packaging.python.org/requirements/) and [this discussion](https://www.reddit.com/r/Python/comments/3uzl2a/setuppy_requirementstxt_or_a_combination/). This is my interpretation it seems that ```requirements.txt``` should be written for development environments, and production (pypi, mostly) dependencies should be handled with ```setuptools```. Let me know if I misinterpreted these sources